website/source/docs/agent/acl-system.html.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: "docs"
+page_title: "ACL System"
+sidebar_current: "docs-agent-acl-system"
+description: |-
+  Consul provides an optional Access Control List (ACL) system which can be used to control access to data and APIs. The ACL system is a Capability-based system that relies on tokens which can have fine grained rules applied to them. It is very similar to AWS IAM in many ways.
+---
+
+-> **1.4.0 and later:** This guide only applies in Consul versions 1.4.0 and later. The documentation for the legacy ACL system is [here](/docs/guides/acl-legacy.html)
+
+# ACL System
+
+Consul provides an optional Access Control List (ACL) system which can be used to control access to data and APIs.
+The ACL is [Capability-based](https://en.wikipedia.org/wiki/Capability-based_security), relying on tokens which
+are associated with policies to determine which fine grained rules can be applied. Consul's capability based
+ACL system is very similar to the design of [AWS IAM](https://aws.amazon.com/iam/).
+
+## ACL System Overview
+
+The ACL system is designed to be easy to use and fast to enforce while providing administrative insight.
+At the highest level, there are two major components to the ACL system:
+
+ * **ACL Policies** - Policies allow the grouping of a set of rules into a logical unit that can be reused and linked with
+ many tokens.
+
+ * **ACL Tokens** - Requests to Consul are authorized by using bearer token. Each ACL token has a public
+ Accessor ID which is used to name a token, and a Secret ID which is used as the bearer token used to
+ make requests to Consul.
+
+ ACL tokens and policies are managed by Consul operators via Consul's
+[ACL API](/api/acl/acl.html), ACL CLI, or systems like
```

Comment:
ACL CLI should link to the acl cli docs.

Reply:
Done!